techniques/css/C41.html

```diff
@@ -51,43 +51,43 @@
 
 <div class="default-grid with-gap leftcol">
 <aside class="box nav-hack sidebar standalone-resource__sidebar">
-    <header class="box-h">Page Contents</header>
+    <header class="box-h">このページの内容</header>
     <div class="box-i">
       <nav aria-label="page contents" class="navtoc">
-        <ul><li><a href="#technique">About this Technique</a></li>
-<li><a href="#description">Description</a></li>
-<li><a href="#examples">Examples</a></li>
-<li><a href="#resources">Related Resources</a></li>
-<li><a href="#related">Related Techniques</a></li>
-<li><a href="#tests">Tests</a></li>
+        <ul><li><a href="#technique">このテクニックについて</a></li>
+<li><a href="#description">解説</a></li>
+<li><a href="#examples">事例</a></li>
+<li><a href="#resources">関連リソース</a></li>
+<li><a href="#related">関連テクニック</a></li>
+<li><a href="#tests">検証</a></li>
 </ul>
       </nav>
     </div>
 </aside>
 
 <main id="main" class="standalone-resource__main">
 		
-<h1><span>Technique C41:</span>Creating a strong focus indicator within the component</h1>
+<h1><span>テクニック C41:</span>コンポーネント内に明るいフォーカスインジケータを作成する</h1>
 
 
 
 
 <section id="technique" class="box">
-	<h2 class="box-h box-h-icon">About this Technique</h2>
+	<h2 class="box-h box-h-icon">このテクニックについて</h2>
 	<div class="box-i">
 
 
   
   <p>
-    This technique relates to
+    このテクニックは
     
-<a href="https://w3c.github.io/wcag/understanding/focus-appearance">2.4.13: Focus Appearance</a>
-(<strong>Sufficient</strong>).</p>
+<a href="https://waic.jp/translations/WCAG22/Understanding/focus-appearance">2.4.13: フォーカスの外観</a>
+(<strong>十分なテクニック</strong>) に関連する。</p>
   
 
 
 
-  <p>This technique applies to <abbr title="Cascading Style Sheets">CSS</abbr>.</p>
+  <p>このテクニックは <abbr title="Cascading Style Sheets">CSS</abbr> に適用する。</p>
```

Comment:
「このテクニックは ... に適用**される**」がよさそうです。(他の PR ではそのように翻訳されているようです。)

```suggestion
  <p>このテクニックは <abbr title="Cascading Style Sheets">CSS</abbr> に適用される。</p>
```